### DISCUSSION TITLE
how do you get by the "Activate User" dialog?

### DISCUSSION THREAD

**Discussion by dovholuknf** [zitadel/zitadel#8057] [General]
I just pulled the compose file and started it up, logged in and it wants me to verify my email when i use `roo@zitadel.localhost`... Uh... Where is this code going to come from??? I honestly tried to read the doc and scan issues but I can't find the answer yet. None of the recently asked discussions seem to have this info? I thought maybe it'd be in the logs, but I don't see it there either. The video guide (from https://zitadel.com/docs/self-hosting/deploy/compose) doesn't show this dialog either. I feel like I missed something silly, but I've tried to follow the instructions a few times and still get the same result. Thx

![image](https://github.com/zitadel/zitadel/assets/46322585/af1a6364-c982-476d-817a-769cc292c0ad)

**Comment by dovholuknf** [comment]
Yeah so, after pulling 2.54.1 i noticed I had somehow used root@zitadel.localhost, not zitadel-admin@zitadel.localhost and i don't know why/where/how I did that... I must have found that from somewhere, but using `zitadel-admin@zitadel.localhost` fixed my foolish issue... thanks :/ my bad!

**Reply by fforootd** [reply]
Hey, great you figured this already!

Thanks for sharing.
